react/components/Footer/components/FooterPaymentFormList.js

```diff
@@ -1,27 +1,17 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import footerList from './footerList'
+import withImage from './withImage'
 
 class FooterPaymentFormItem extends Component {
```

Comment:
Since this component (and the other ones) don't utilize state anymore, you can just export them as SFC